pkg/image/external/external.go

```diff
@@ -92,6 +94,8 @@ func GetExternalImages(rancherVersion string, externalData map[string]interface{
 		// Registries don't allow "+", so image names will have these substituted.
 		upgradeImage := fmt.Sprintf("rancher/%s-upgrade:%s", source, strings.Replace(release, "+", "-", -1))
 		externalImagesMap[upgradeImage] = true
+		systemAgentInstallerImage := fmt.Sprintf("%s%s:%s", settings.SystemAgentInstallerImage.Get(), source, strings.Replace(release, "+", "-", -1))
```

Comment:
I suspect this is doing the same as what we do [here](https://github.com/rancher/rancher/blob/9de50354f56f69d56f84d324db62db624832486f/scripts/package#L27-L39). If that's the case, you should remove it from package since having it in the Go code is preferable.

Reply:
I did not realize we had that in scripts/package and thought it was overlooked so I added it. I will update scripts/package

Reply:
On second thought, I am going to move the scripts/package task out to a secondary PR as I want to have the time to ensure that there isn't a regression in the way the image lists are built shortly before 2.6.5 is set to be released.

Reply:
https://github.com/rancher/rancher/issues/37596